test/lib/Create2.t.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+
+import {Create2} from "src/lib/Create2.sol";
+
+contract ComputeCreate2Address is Test {
+  function test_GenerateCreate2Address() external {
+    bytes32 salt = bytes32(uint256(31_415));
+    bytes memory initcodeHash = abi.encode(0x6080);
```

Comment:
nit, but let's make this a bit more realistic by actually hasing, it's a bit confusing like this I think, though I get it doesn't matter
```suggestion
    bytes memory initcodeHash = keccak256(abi.encode(0x6080));
```

Reply:
https://github.com/ScopeLift/optimizoors-perpetual-protocol/pull/19/commits/b68701f8f5ed0faefd3c398f23d7c8b56c4c2bfb I had to also convert to bytes as that is what computeCreate2Address is expecting

Reply:
oh I see, that's because that method is expecting the initcode, not the initcode hash, so really the suggestion should be to change the var name from `initcodeHash` to just `initcode`. Because `Create2.computeCreate2Address` does the hashing

Reply:
I just pushed 476e18f to do the rename